std/meta.d

```diff
@@ -269,8 +249,8 @@ if (!isAggregateType!T || is(Unqual!T == T))
 }
 
 /**
- * Returns the index of the first occurrence of type T in the
- * sequence of zero or more types TList.
+ * Returns the index of the first occurrence of T in the
```

Comment:
Why remove the word "type" here? I also don't know what these documentation changes have to do with the changes relating to the unittest above.

Reply:
Because staticIndexOf can find symbols or values, not just types.
The unittest is part of the modules documentation too, but I can split it out into another pull if you prefer.

Reply:
But `T` there is a type, it's not an alias template parameter.

Reply:
For that overload, but T is also an alias parameter for the other overload:
https://github.com/dlang/phobos/blob/8308cef30bdd843d67ef22f5feb64aadbe0ca35f/std/meta.d#L261

Reply:
Ah, I see.